src/terminal/Screen.zig

```diff
@@ -2591,13 +2591,24 @@ pub fn selectOutput(self: *Screen, pin: Pin) ?Selection {
     const start: Pin = boundary: {
         var it = pin.rowIterator(.left_up, null);
         var it_prev = pin;
+
+        // First, iterate until we find the first line of command output
         while (it.next()) |p| {
+            it_prev = p;
             const row = p.rowAndCell().row;
-            switch (row.semantic_prompt) {
-                .command => break :boundary p,
-                else => {},
+            if (row.semantic_prompt == .command) {
```

Comment:
We should use an exhaustive `switch` here. Even when we're matching only one enum element, I prefer to use an exhaustive switch so that the compiler can error and tell us all the places a switch is used when we add an element. This makes it easy to consider if we need to change our behavior.

(I know the old code didn't use an exhaustive switch, that was a mistake)

Reply:
Done :)